.github/workflows/agent-charm-release-edge.yml

```
@@ -0,0 +1,27 @@
+name: Release testflinger-agent charm to latest/edge
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - agent/charms/testflinger-agent-charm/**
```

I suggest to run this action also when there are updates to this CI file

---

I was going to say we could just use workflow dispatch if we really need to trigger a build, but I can see some value in also using this as a test of any change that might get made to the workflow file in the future. Good idea!
Fixed

---

@plars did you implement it elsewhere or just forgot to push it here?
```suggestion
    paths:
      - agent/charms/testflinger-agent-charm/**
      - .github/workflows/agent-charm-release-edge.yml
```

---

@nancyc12 @nadzyah I could swear I pushed it... guess not it was sitting in my local branch feeling like I forgot about it :)